src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/integration/SetupAssistant.kt

```diff
@@ -114,6 +116,7 @@ class SetupAssistant(
     appointmentDeliveryRepository.deleteAll()
     appointmentRepository.deleteAll()
 
+    referralDetailsRepository.deleteAll()
```

Comment:
_future-proofing, not a blocker_
To avoid remembering cleaning individual tables, I wonder if we can do a checkpoint/snapshot in the beginning and rollback to it at the end, or do a DB reset?